pkg/fission-cli/cmd/spec/list.go

```diff
@@ -60,56 +60,101 @@ func (opts *ListSubCommand) run(input cli.Input) error {
 		deployID = fr.DeploymentConfig.UID
 	}
 
-	allfn, err := getAllFunctions(opts.Client())
+	_, currentNS, err := util.GetResourceNamespace(input, flagkey.NamespaceEnvironment)
+	if err != nil {
+		return fv1.AggregateValidationErrors("Environment", err)
+	}
+
+	var allfn []fv1.Function
+	if input.Bool(flagkey.AllNamespaces) {
```

Comment:
This if&else block is across function. Should we define variable and use it across `getAll...` functions ?

Reply:
Have refactored it to use single if-else